pyriemann/embedding.py

```diff
@@ -1,12 +1,17 @@
 """Embedding covariance matrices via manifold learning techniques."""
 
 import numpy as np
-from sklearn.base import BaseEstimator
+from scipy.linalg import solve, eigh
+from scipy.sparse import csr_matrix
+
+from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.manifold import spectral_embedding
+
+from .utils.kernel import kernel
 from .utils.distance import pairwise_distance
 
 
-class Embedding(BaseEstimator):
```

Comment:
Embedding class was released in v0.2.5 so technically you should deprecate here and not break silently public API.